### PR TITLE
Revert "Fixed incorrect parameters provided to Command#yield-control"

### PR DESCRIPTION
**Pull request by mandrigin**
This reverts commit ac1de8c690c26696965979ad5f7105dfac15726e because it causes a regression making sending any assets in a chat impossible.

status: ready

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

**Comment by mandrigin** [comment]
PS: this should pass e2e with 100% before merging.

**Review by jeluard — APPROVED** [review]

**Review by rasom — APPROVED** [review]

**Review by antdanchenko — APPROVED** [review]

**Comment by oskarth** [comment]
Thanks @mandrigin. Did we or are we planning a post mortem for this? Few questions that stick out right now re offending PR: https://github.com/status-im/status-react/pull/7283

1) Why was this merged when automated tests were failed? @jeluard 

2) Why was `skip-manual-qa` used for something that impacted running code? @jeluard 

3) Why aren't automated tests blocking yet? We decided before the break we'd make them so, and removed the remaining blockers for it. @antdanchenko  @annadanchenko

**Comment by jeluard** [comment]
I didn't notice e2e tests didn't pass as I was confused by all the green checks everywhere and nothing happened after some time. Using `skip-manual-qa` was a mistake.

I would propose to:

* block PR while not all checks have been performed
* replace `skip-manual-qa` with `manual-sanity-check`
* block PR and re-run e2e after each force push

What about running e2e as soon as the PR is opened? It provides developers insights and doesn't need the review before it can be performed.

**Comment by antdanchenko** [comment]
@oskarth
Automated tests are blocking PRs by setting failed check status to a commit

Actual workflow is:

- After creation fo a PR it is blocked by missing approvals 
- PR is unblocked after getting approvals
- PR is moved into 'TO TEST`  section by Autobot and automated tests are triggered if all other checks are green
- Tests are setting `pending status` for the commit on start 
- In the end of test run status is changed to failed or passed

**Comment by pedropombeiro** [comment]
@oskarth I'd like to work to get a Probot to block the PR from the moment it is created due to lack of e2e tests (with the possible exception of a pre-defined filter for non-important files like .gitignore or docs folder) so we can avoid this gap between steps 2 and 5 above.

**Comment by antdanchenko** [comment]
@oskarth #7283 was never moved into `TO TEST` section due to `skip-manual-qa` label so tests were not triggered and the status hasn't been set

We can set the blocked status right after creation of a PR to prevent such issues
Or we can stop using `skip-manual-qa` label on PRs which are changing the product code

@jeluard 
Replacing `skip-manual-qa` with `manual-sanity-check` is not a solution

`skip-manual-qa` is used in PRs which are changing e2e tests or infrastructure, changes which not impact product

We shouldn't skip testing of PRs which are changing the product behavior
Also we should add the description of the implemented feature in order not to test possible regressions but to test the actual feature and areas which might by affected by the change.

**Comment by antdanchenko** [comment]
@jeluard
We can't allow running e2e tests after each force push, we've tried that a year ago and got 24 hours queue
We are running automated tests only when a PR is ready for testing

**Pull request merged**
